topics/computer-algebra/index.md

```diff
@@ -0,0 +1,10 @@
+---
+aliases: computeralgebra, symbolic-algebra, symbolicalgebra
+display_name: Computer algebra
+short_description: Computer algebra libraries are used for manipulations with mathematical expressions.
+topic: computer-algebra
+wikipedia_url: https://en.wikipedia.org/wiki/Computer_algebra
+---
+Computer (symbolic) algebra systems and libraries are used for manipulations with symbolic mathematical expressions.
+Usually, those libraries are capable of some calculus operations (differentiation, integration, finding the limit, etc.), numeric manipulations.
```

Comment:
```suggestion
Usually, those libraries are capable of some calculus operations (differentiation, integration, finding the limit, etc.) and numeric manipulations.
```

seems like an `and` works better here than a comma. WDYT?

Reply:
I guess you're right